python_modules/dagster/dagster/core/events/pipeline.py

```diff
@@ -0,0 +1,59 @@
+from enum import Enum
+from collections import namedtuple
+
+from dagster import check
+from dagster.core.events.logging import EventType
+from dagster.core.execution import SystemPipelineExecutionContext
+
+
+class PipelineEventType(Enum):
+    START = 'START'
+    SUCCESS = 'SUCCESS'
+    FAILURE = 'FAILURE'
+
+    PROCESS_START = 'PROCESS_START'
+    PROCESS_STARTED = 'PROCESS_STARTED'
+
+
+class PipelineEvent(namedtuple('_PipelineEvent', 'event_type_value pipeline_name')):
```

Comment:
Add a comment indicating that this is part of the DagsterEvent union (which should exist somewhere if it doesn't)